src/js/views/patients/schedule/schedule_views.js

```diff
@@ -116,18 +116,16 @@ const DayItemView = View.extend({
   templateContext() {
     const state = this.model.getState();
 
-    const isReduced = this.state.get('isReduced');
-
     return {
       isOverdue: this.model.isOverdue(),
       state: state.get('name'),
       stateOptions: state.get('options'),
       patient: this.model.getPatient().attributes,
       form: this.model.getForm(),
-      isSelected: !isReduced && this.state.isSelected(this.model),
+      isSelected: !this.isReduced && this.state.isSelected(this.model),
```

Comment:
can `selected` still be set if the user is reduced now?  I'd assume not.  I'd be this is a missing branch in coverage

Reply:
If I remove the `!this.isRedcued` check, a `"this.state.isSelected is not a function"` console error is thrown and the schedule doesn't load any actions.

The normal schedule app uses this custom state model where the `isSelected()` function is used from: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/patients/schedule/schedule_state.js.

Since we don't use that state model for the reduced schedule, it's looking for that function and not finding it.

Reply:
ah right.. no state model

Reply:
I forgot this view is in both places